### PR TITLE
Updated terminologies for chaos agent, workflow and charts

### PR DESCRIPTION
**Pull request by Saranya-jena**
Signed-off-by: Saranya-jena <saranya.jena@harness.io>

**What this PR does / why we need it**:
- Chaos Agents → Chaos Delegates
- Chaos Workflows → Chaos Scenarios
- Chaos Charts → Chaos Experiments

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/troubleshooting.md [hunk 1]
```diff
@@ -86,20 +89,20 @@ mkdir <hub_name>
...
-kubectl cp <location to chaos-chart/hub directory> <namespace>/<litmusportal-server-pod-name>:</tmp/version/<project_id>/<hub_name> -c graphql-server
+kubectl cp <location to chaos-experiment/hub directory> <namespace>/<litmusportal-server-pod-name>:</tmp/version/<project_id>/<hub_name> -c graphql-server
 ```
 
+Example:
 
-Example: 
 ```
-kubectl cp /home/amitkrdas/Chaos-Charts/chaos-charts/  litmus/litmusportal-server-6df9c5895d-57xx7:/tmp/version/686c1da2-da9c-4029-9c6a-528a9455a3b3/"Litmus ChaosHub" -c graphql-server 
+kubectl cp /home/amitkrdas/Chaos-Charts/chaos-experiments/  litmus/litmusportal-server-6df9c5895d-57xx7:/tmp/version/686c1da2-da9c-4029-9c6a-528a9455a3b3/"Litmus ChaosHub" -c graphql-server
```
Comment (SarthakJain26) [inline website/docs/troubleshooting.md]: Is this link correct?

File: website/docs/architecture/chaos-observability-flow-analytics.md [hunk 2]
```diff
@@ -10,13 +10,14 @@ sidebar_label: Analytics
...
 ## Observability Flow for Analytics
-1. In the Chaos Execution Plane, the ChaosEngine Details and ChaosResult are fetched by the Chaos Agent.
-2. Chaos Agent then forwards them to the Backend Server in the Chaos Control Plane and later they get stored into the Database.
-3. User specifies the Chaos Workflow Schedule for which the workflow statistics and information is to be fetched as an input in the ChaosCenter.
-4. The request for the workflow statistics and information is received by the Backend Server.
-5. Backend Server queries the Database for the details of past Workflow Runs.
-6. Aggregated workflow statistics based on the ChaosResult verdict and probe success percentage are fetched from the Database by Backend Server. 
-7. Workflow statistics and information are forwarded to ChaosCenter by Backend Server.
+
+1. In the Chaos Execution Plane, the ChaosEngine Details and ChaosResult are fetched by the Chaos Delagate.
```
Comment (Jonsy13) [inline website/docs/architecture/chaos-observability-flow-analytics.md]: Please check in other places too.

```suggestion
1. In the Chaos Execution Plane, the ChaosEngine Details and ChaosResult are fetched by the Chaos Delegate.
```

File: website/docs/concepts/chaos-workflow.md [hunk 3]
```diff
@@ -269,7 +269,7 @@ spec:
   schedule: 10 0-23 * * *
   concurrencyPolicy: Forbid
   startingDeadlineSeconds: 0
-  workflowSpec:
+ chaos scenarioSpec:
```
Comment (Jonsy13) [inline website/docs/concepts/chaos-workflow.md]: Please revert this

File: website/docs/concepts/chaoshub.md [hunk 4]
```diff
@@ -92,13 +92,13 @@ To remove a ChaosHub from a project, you can select the **Disconnect Hub** optio
 
 ## Summary
 
-ChaosHubs are basically a collection of different clones of the Chaos-Charts repository, which consists of a variety of experiments and pre-defined workflows. You can use a ChaosHub to construct a custom chaos workflow and tune it according to the use-case. These ChaosHubs can also be synced with the latest changes. New experiments and pre-defined workflows can also be added in the repository which can be directly used in the ChaosCenter.
+ChaosHubs are basically a collection of different clones of the Chaos-Charts repository, which consists of a variety of experiments and pre-defined haos scenarios. You can use a ChaosHub to construct a custom chaos scenarios and tune it according to the use-case. These ChaosHubs can also be synced with the latest changes. New experiments and pre-defined chaos scenarios can also be added in the repository which can be directly used in the ChaosCenter.
```
Comment (Jonsy13) [inline website/docs/concepts/chaoshub.md]:
```suggestion
ChaosHubs are basically a collection of different clones of the Chaos-Charts repository, which consists of a variety of experiments and pre-defined chaos scenarios. You can use a ChaosHub to construct a custom chaos scenarios and tune it according to the use-case. These ChaosHubs can also be synced with the latest changes. New experiments and pre-defined chaos scenarios can also be added in the repository which can be directly used in the ChaosCenter.
```

File: website/docs/concepts/chaos-workflow.md [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
 
 - [ChaosCenter](../getting-started/resources.md#chaoscenter)
-- [ChaosAgent](../getting-started/resources.md#chaosagents)
+- [Chaos Delegate](../getting-started/resources.md#chaosagents)
 - [Chaos Experiment CR](chaos-experiment.md)
 - [ChaosEngine CR](chaos-engine.md)
 - [Probes](probes.md)
 
-## How do we define and execute a workflow?
+## How do we define and execute a ?
```
Comment (amityt) [inline website/docs/concepts/chaos-workflow.md]: How do we define and execute a chaos workflow*

File: website/docs/faq.md [hunk 6]
```diff
@@ -48,15 +48,15 @@ Yes, you can run the ansible experiments outside of the k8s cluster which is doc
...
 
-### Can I create workflows without using dashboard?
+### Can I create chaos scenarios without using dashboard?
 
 Currently, you can’t. But we are working on it. Shortly we will publish samples for doing this via API/SDK and litmusctl.
 
 ### Does Litmusctl support actions that are currently performed from the portal dashboard?
 
-For now, you can create agents and projects, also you can get the agents and project details by using litmusctl. To know more about litmusctl please refer to the [documentation of litmusctl](https://github.com/litmuschaos/litmusctl/blob/master/Usage.md).
+For now, you can createchaos delegates and projects, also you can get thechaos delegates and project details by using litmusctl. To know more about litmusctl please refer to the [documentation of litmusctl](https://github.com/litmuschaos/litmusctl/blob/master/Usage.md).
```
Comment (amityt) [inline website/docs/faq.md]: Space is missing

File: website/docs/faq.md [hunk 7]
```diff
@@ -48,15 +48,15 @@ Yes, you can run the ansible experiments outside of the k8s cluster which is doc
...
 
-### What is the minimum system requirement to run Portal and agent together?
+### What is the minimum system requirement to run Portal andchaos delegate together?
```
Comment (amityt) [inline website/docs/faq.md]: space between "and chaos"

File: website/docs/getting-started/run-your-first-workflow.md [hunk 8]
```diff
@@ -117,7 +117,7 @@ And with that you have successfully scheduled your first Chaos Workflow with Lit
    podtato-head-chaos-1622018315-80657052      0/2     Completed           0          79s
    podtato-head-pod-delete-chaosx55vh-runner   1/1     Running             0          25s
    subscriber-958948965-qbx29                  1/1     Running             0          6h16m
-   workflow-controller-78fc7b6c6-w82m7         1/1     Running             0          6h16m
+   Chaos Scenario-controller-78fc7b6c6-w82m7         1/1     Running             0          6h16m
```
Comment (amityt) [inline website/docs/getting-started/run-your-first-workflow.md]: This can be in small letters, representing a pod-name

File: website/docs/faq.md [hunk 9]
```diff
@@ -19,14 +19,14 @@ Yes, with the help of github-chaos-action we can automate the chaos execution on
...
-Other than that there is another key in the control plane’s configmap `litmus-portal-admin-config` called `AgentScope`, this is given to allow users to restrict access to the litmus self-agent components self-agent is the agent for your control plane cluster (exporter, operator etc), you can use both of them in a way to give access as per the requirement.
-The above holds for the control plane and self agent, for the external agents which can be connected using the litmusctl CLI you can provide the scope of the agent while using the utility to connect your other cluster to the control plane with access to just a single namespace or cluster-wide access.
-Using a combination of AgentScope: cluster and `PORTAL_SCOPE` env set to cluster would give you cluster-admin privileges to inject chaos on all namespaces where the control plane/portal is installed. For external agents just selecting the scope of installation as cluster would be sufficient via litmusctl.
+That basically restricts portal installation along with itschaos delegate to a single namespace and the chaos operator, exporter all get installed in a single namespace and can only perform and monitor chaos in that namespace.
```
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
That basically restricts portal installation along with its chaos delegate to a single namespace and the chaos operator, exporter all get installed in a single namespace and can only perform and monitor chaos in that namespace.
```

File: website/docs/faq.md [hunk 10]
```diff
@@ -19,14 +19,14 @@ Yes, with the help of github-chaos-action we can automate the chaos execution on
...
+Other than that there is another key in the control plane’s configmap `litmus-portal-admin-config` called `AgentScope`, this is given to allow users to restrict access to the litmus self chaos delegate components self chaos delegate is thechaos delegate for your control plane cluster (exporter, operator etc), you can use both of them in a way to give access as per the requirement.
```
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
Other than that there is another key in the control plane’s configmap `litmus-portal-admin-config` called `AgentScope`, this is given to allow users to restrict access to the litmus self chaos delegate components self chaos delegate is the chaos delegate for your control plane cluster (exporter, operator etc), you can use both of them in a way to give access as per the requirement.
```

**Review by amityt — CHANGES_REQUESTED** [review]

File: website/docs/faq.md [hunk 11]
```diff
@@ -19,14 +19,14 @@ Yes, with the help of github-chaos-action we can automate the chaos execution on
...
+The above holds for the control plane and selfchaos delegate, for the externalchaos delegates which can be connected using the litmusctl CLI you can provide the scope of thechaos delegate while using the utility to connect your other cluster to the control plane with access to just a single namespace or cluster-wide access.
```
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
The above holds for the control plane and self chaos delegate, for the external chaos delegates which can be connected using the litmusctl CLI you can provide the scope of the chaos delegate while using the utility to connect your other cluster to the control plane with access to just a single namespace or cluster-wide access.
```

File: website/docs/faq.md [hunk 12]
```diff
@@ -19,14 +19,14 @@ Yes, with the help of github-chaos-action we can automate the chaos execution on
...
+Using a combination of AgentScope: cluster and `PORTAL_SCOPE` env set to cluster would give you cluster-admin privileges to inject chaos on all namespaces where the control plane/portal is installed. For externalchaos delegates just selecting the scope of installation as cluster would be sufficient via litmusctl.
```
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
Using a combination of AgentScope: cluster and `PORTAL_SCOPE` env set to cluster would give you cluster-admin privileges to inject chaos on all namespaces where the control plane/portal is installed. For external chaos delegates just selecting the scope of installation as cluster would be sufficient via litmusctl.
```

File: website/docs/faq.md (hunk 6 above)
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
For now, you can create chaos delegates and projects, also you can get the chaos delegates and project details by using litmusctl. To know more about litmusctl please refer to the [documentation of litmusctl](https://github.com/litmuschaos/litmusctl/blob/master/Usage.md).
```

File: website/docs/faq.md (hunk 7 above)
Comment (Jonsy13) [inline website/docs/faq.md]:
```suggestion
### What is the minimum system requirement to run Portal and chaos delegate together?
```

File: website/docs/getting-started/resources.md [hunk 13]
```diff
@@ -36,34 +36,34 @@ The ChaosCenter gives you access to a plethora of features, the major ones inclu
...
+As part of the Litmus installation, a Self Chaos Delegate would be registered as a default Chaos Delegate in the ChaosCenter. The same cluster where Litmus is installed is chosen as the Self Chaos Delegate by the installer. From the ChaosCenter you can now induce chaos into this Self Chaos Delegate and observe the results.
 
-Since the ChaosCenter is Cross Cloud, you can connect multiple external Kubernetes agents to the same with the help of the command line utility [litmusctl](../litmusctl/installation.md). Once connected you can manage, monitor, observe and induce chaos from the ChaosCenter to the respective ChaosAgents.
+Since the ChaosCenter is Cross Cloud, you can connect multiple external Chaos Deleagtes to the same with the help of the command line utility [litmusctl](../litmusctl/installation.md). Once connected you can manage, monitor, observe and induce chaos from the ChaosCenter to the respective Chaos Delegates.
```
Comment (Jonsy13) [inline website/docs/getting-started/resources.md]:
```suggestion
Since the ChaosCenter is Cross Cloud, you can connect multiple external Chaos Delegates to the same with the help of the command line utility [litmusctl](../litmusctl/installation.md). Once connected you can manage, monitor, observe and induce chaos from the ChaosCenter to the respective Chaos Delegates.
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 14]
```diff
@@ -1,16 +1,16 @@
 ---
 id: run-your-first-workflow
-title: Run your First Chaos Workflow in 5 minutes
-sidebar_label: Run Your First Workflow
+title: Run your First Chaos Scenario in 5 minutes
+sidebar_label: Run Your FirstChaos Scenario
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
sidebar_label: Run Your First Chaos Scenario
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 15]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
 
 ---
 
-## Schedule your First Workflow
+## Schedule your FirstChaos Scenario
 
-### Select the podtato-head predefined workflow
+### Select the podtato-head predefinedChaos Scenario
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
### Select the podtato-head predefined Chaos Scenario
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 16]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
 
-1. Click on <span style={{color: '#5B44BA'}}>**Schedule a Workflow**</span> from the ChaosCenter Homepage or from the top right button in the **Litmus Workflows** tab.
+1. Click on <span style={{color: '#5B44BA'}}>**Schedule aChaos Scenario**</span> from the ChaosCenter Homepage or from the top right button in the **Litmus Workflows** tab.
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
1. Click on <span style={{color: '#5B44BA'}}>**Schedule a Chaos Scenario**</span> from the ChaosCenter Homepage or from the top right button in the **Litmus Workflows** tab.
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 17]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
+5. View the visualization of the Litmus Chaos Scenario you are about to execute. This step also allows for you to edit or modify the YAML/tunable if required. We would just stick with the default configurations for now.
 
-   <img src={require("../assets/getting-started/run-your-first-workflow/podtato-head-tune-workflow.png").default} alt="Visualize Podtato Head Workflow" />
+   <img src={require("../assets/getting-started/run-your-first-workflow/podtato-head-tune-workflow.png").default} alt="Visualize Podtato Head Chaos Scenario" />
 
-   > By default in the Podtato Head Workflow Template the steps to gracefully delete the Chaos Resources (`revert-chaos`) and also the Podtato Head application (`delete-application`) are present.
+   > By default in the Podtato Head Chaos Scenario Template the steps to gracefully delete the Chaos Resources (`revert-chaos`) and also the Podtato Head application (`delete-application`) are present.
 
-### Assign Weights to the Workflow Experiments
+### Assign Weights to theChaos Scenario Experiments
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
### Assign Weights to the Chaos Scenario Experiments
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 18]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
    <img src={require("../assets/getting-started/run-your-first-workflow/select-podtato-head.png").default} alt="Select Podtato Head" />
 
-4. View the workflow details in the **Workflow Settings**, you can modify the name and description of the workflow to suit your needs.
+4. View the Chaos Scenario details in the **Chaos Scenario Settings**, you can modify the name and description of the Chaos Scenario to suit your needs.
 
-   <img src={require("../assets/getting-started/run-your-first-workflow/workflow-settings-podtato-head.png").default} alt="Modify Details of Podtato Head Workflow" />
+   <img src={require("../assets/getting-started/run-your-first-workflow/workflow-settings-podtato-head.png").default} alt="Modify Details of Podtato Head Chaos Scenario" />
 
-### Simulate the Workflow Steps Visualization
+### Simulate theChaos Scenario Steps Visualization
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
### Simulate the Chaos Scenario Steps Visualization
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 19]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
+### Assign Weights to theChaos Scenario Experiments
 
-6. Assign weights to the chaos experiments that are part of the workflow using the slider. This is typically used when there are multiple experiments as part of a workflow. These weights influence the **Resilience Score** calculation for the chaos workflow.
+6. Assign weights to the chaos experiments that are part of theChaos Scenario using the slider. This is typically used when there are multiple experiments as part of a Chaos Scenario. These weights influence the **Resilience Score** calculation for the Chaos Scenario.
 
-   <img src={require("../assets/getting-started/run-your-first-workflow/adjust-weights-for-podtato-head.png").default} alt="Assign Weights for Workflow Experiments" />
+   <img src={require("../assets/getting-started/run-your-first-workflow/adjust-weights-for-podtato-head.png").default} alt="Assign Weights for Chaos Scenario Experiments" />
 
-### Schedule the Podtato-Head Workflow
+### Schedule the Podtato-HeadChaos Scenario
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
### Schedule the Podtato-Head Chaos Scenario
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 20]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
-7. Schedule the Litmus Workflow for immediate and one-time execution by selecting the **Schedule Now** option
+7. Schedule the Litmus Chaos Scenario for immediate and one-time execution by selecting the **Schedule Now** option
 
-   <img src={require("../assets/getting-started/run-your-first-workflow/choose-schedule.png").default} alt="Choose a Schedule for Workflow Execution" />
+   <img src={require("../assets/getting-started/run-your-first-workflow/choose-schedule.png").default} alt="Choose a Schedule for Chaos Scenario Execution" />
 
 8. Verify and click on **Finish** to start the Chaos Injection
 
-   <img src={require("../assets/getting-started/run-your-first-workflow/verify-and-commit-podtato-head.png").default} alt="Verify and Schedule the Podtato Head Workflow" />
+   <img src={require("../assets/getting-started/run-your-first-workflow/verify-and-commit-podtato-head.png").default} alt="Verify and Schedule the Podtato HeadChaos Scenario" />
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
   <img src={require("../assets/getting-started/run-your-first-workflow/verify-and-commit-podtato-head.png").default} alt="Verify and Schedule the Podtato Head Chaos Scenario" />
```

File: website/docs/getting-started/run-your-first-workflow.md [hunk 21]
```diff
@@ -24,71 +24,71 @@ In this scenario we will execute a pod-delete fault on a sample micro-service ap
...
 
-<img src={require("../assets/getting-started/run-your-first-workflow/scheduled-successfully.png").default} alt="Podtato Head Workflow Scheduled Successfully" />
+<img src={require("../assets/getting-started/run-your-first-workflow/scheduled-successfully.png").default} alt="Podtato Head Chaos Scenario Scheduled Successfully" />
 
 ---
 
 ## Visualize and Analyze
 
-1. To check the current progress of the Podtato-Head workflow, view the status of the Workflow from the **Litmus Workflows** Tab.
+1. To check the current progress of the Podtato-HeadChaos Scenario, view the status of theChaos Scenario from the **Litmus Chaos Scenarios** Tab.
```
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]:
```suggestion
1. To check the current progress of the Podtato-Head Chaos Scenario, view the status of the Chaos Scenario from the **Litmus Chaos Scenarios** Tab.
```

File: website/docs/getting-started/run-your-first-workflow.md (hunk 8 above)
Comment (Jonsy13) [inline website/docs/getting-started/run-your-first-workflow.md]: We can revert this change

File: website/docs/troubleshooting.md [hunk 22]
```diff
@@ -35,16 +35,16 @@ Just run the following command:
 
 Make sure to update the namespace and mongo pod name according to your setup, the rest should remain the same. This command will update the password to `litmus`.
 
-### While uninstalling Litmus portal using helm, some components like subscriber, exporter, event, workflows, etc are not removed.
+### While uninstalling Litmus portal using helm, some components like subscriber, exporter, event, chaos scenatios, etc are not removed.
 
-These are agent components, which are launched by the control plane server, so first disconnect the agent from the portal then uninstall the portal using helm.
+These arechaos delegatecomponents, which are launched by the control plane server, so first disconnect thechaos delegatefrom the portal then uninstall the portal using helm.
```
Comment (Jonsy13) [inline website/docs/troubleshooting.md]:
```suggestion
These are chaos delegate components, which are launched by the control plane server, so first disconnect the chaos delegate from the portal then uninstall the portal using helm.
```

File: website/docs/user-guides/schedule-workflow.md [hunk 23]
```diff
@@ -1,86 +1,84 @@
...
-   With revert step enabled, a new functionality called `podGC` is also added which deletes the workflow pods after the completion of workflow as part of the clean-up process.
+1. **Chaos Scenario Visualization** : This feature allows you to visualize the chaos scenario even before scheduling it.
+   This gives a brief information related to the structure of chaos scenario <i>i.e</i> if the experiments are present in serial or parallel way.
+2. **Chaos Scenario Table** : This table contains the list of experiments present in the chaos scenario . It also consists of some valuable information related to the target applications.
+3. **Add Experiment** : If you have selected `Create a new chaos scenario using experiments from MyHub` in Choose a Chaos Scenario step, you can see a `Add a new experiment` button, this will allow you to add more experiments to the chaos scenario .
+4. **Edit Chaos Scenario ** : With this option, you can view and make changes in the chaos scenario manifest with a YAML editor.
+5. **Revert Chaos** : For custom chaos scenario s, you can now enable or disable the revert step from the portal.
```
Comment (Jonsy13) [inline website/docs/user-guides/schedule-workflow.md]:
```suggestion
5. **Revert Chaos** : For custom chaos scenarios, you can now enable or disable the revert step from the portal.
```

**Review by Jonsy13 — CHANGES_REQUESTED** [review]
Have left minor comments PTAL

**Review by amityt — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]
